assets/src/dashboard/components/pageStructure/index.js

```diff
@@ -79,8 +84,7 @@ export const LeftRailContainer = styled.nav.attrs({
   top: 0;
   bottom: 0;
   width: ${DASHBOARD_LEFT_NAV_WIDTH}px;
-  background: ${({ theme }) => theme.DEPRECATED_THEME.colors.white};
-  border-right: ${({ theme }) => theme.DEPRECATED_THEME.borders.gray50};
+  background: ${({ theme }) => theme.colors.standard.white};
```

Comment:
theme.colors.bg.primary please, just incase we need to have a light/dark toggle someday.